src/components/article.js

```diff
@@ -51,27 +56,76 @@ function getAvatarUrl(user){
             })
         },[]);
 
+       
+        const HandleUpVotes = () => {
+            setIsLoading(true);
+            setVoteCount(+1)
+            setErr(null)
+            fetch(`https://hannybees-news-app.herokuapp.com/api/articles${articleIdRoute}`, {
+  method: 'PATCH',
+  body: JSON.stringify({
+    inc_votes: voteCount,
+  }),
+  headers: {
+    'Content-type': 'application/json; charset=UTF-8',
+  },
+}).then((response) => response.json())
+  .then((json) => setArticleData(json.article)).catch((err)=>{
+    setVoteCount(-1)
+    setErr('Something went wrong, please try again.')
+  });
+  
+        }
+    
+                
+        const HandleDownVotes = () => {
+            setIsLoading(true);
+            setVoteCount(-1)
+            setErr(null)
+            fetch(`https://hannybees-news-app.herokuapp.com/api/articles${articleIdRoute}`, {
+  method: 'PATCH',
+  body: JSON.stringify({
+    inc_votes: voteCount,
+  }),
+  headers: {
+    'Content-type': 'application/json; charset=UTF-8',
+  },
+}).then((response) => response.json())
+  .then((json) => setArticleData(json.article)).catch((err)=>{
+    setVoteCount(+1)
+    setErr('Something went wrong, please try again.')
+  })
   
+        }  
+
```

Comment:
Feel like there's a lot going on on this page! Would be better to change the comments view in its own component and vote buttons in their own component. Each with their own useEffect if needed.